azure-proto-compute/VersionOverrides/AvailabilitySetsApiVersions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using azure_proto_core;
+
+namespace azure_proto_compute
+{
+    public class AvailabilitySetsApiVersions : ApiVersionsBase
+    {
+        public static readonly AvailabilitySetsApiVersions V2020_05_01 = new AvailabilitySetsApiVersions("2020-05-01");
+        public static readonly AvailabilitySetsApiVersions V2019_12_01 = new AvailabilitySetsApiVersions("2019-12-01");
+        public static readonly AvailabilitySetsApiVersions Default = V2020_05_01;
+
+        private AvailabilitySetsApiVersions(string value) : base(value) { }
+
+        public static implicit operator string(AvailabilitySetsApiVersions version)
```

Comment:
Is this needed? Does the operator on the base get picked up?

Reply:
I couldn't find a way to get implicit to be inherited from base, if you know of a way I would prefer that, but from everything i could find you must implement it in each concrete class.  This will be auto generated so shouldn't be a big deal